repobee_sanitizer/sanitizer.py

```diff
@@ -111,7 +114,11 @@ def command(self, api) -> Optional[plug.Result]:
                 name="sanitize-file", msg=msg, status=plug.Status.ERROR,
             )
 
-        result = _sanitize.sanitize_file(self.infile)
+        result = (
+            _sanitize.strip_file(self.infile)
+            if self.strip
+            else _sanitize.sanitize_file(self.infile)
+        )
```

Comment:
With the changes I've suggested, this should in the end look something like this:

```suggestion
        relpath = _fileutils.create_relpath(self.infile.absolute(), basedir=pathlib.Path("."))
        result = _sanitize.sanitize_file(relpath, strip=True)
```